src/components/molecule_3d.jsx

```diff
@@ -241,6 +254,7 @@ class Molecule3d extends React.Component {
     return (
       <div
         className="molecule-3d"
+        id="m3d4reactviewer"
```

Comment:
Did you accidentally commit this?  Adding an id will prevent you from using 2 instances on the same page.